conmon-rs/server/src/config.rs

```diff
@@ -79,6 +89,14 @@ impl Config {
             bail!("runtime path '{}' does not exist", self.runtime().display())
         }
 
+        if let Some(rr) = self.runtime_root() {
+            if rr.exists() {
+                fs::create_dir_all(rr)?;
```

Comment:
Why do we have to create the dir if it already exist?

Reply:
because I write buggy code :sweat_smile: good catch